Text-Hoborg/lib/Text/Hoborg/Manuel.pod

```diff
@@ -90,7 +90,7 @@ reckon now I'm a loyal Republic
 citizen, ain't I? 
 
 So. It was early winter, but the weather was still sunny and mild. I was walking
-in the docks, as I was wont to do. The air was clear and I could see ships
+in the docks, as I was accustomed to do. The air was clear and I could see ships
 docking and leaving,
```

Comment:
I do like "wont" here https://www.merriam-webster.com/dictionary/wont It gives it a certain old feel. In fact, it means the same.